notebooks/kaggle/store-sales-time-series-forecasting/experiments/LightGBM.ipynb

```diff
@@ -574,9 +227,7 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "bc198ee0",
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Here I have a question: how to pass 'categorical_feature' parameter to the PositiveRegressor(lgb.LGBMRegressor)?\n",
```

Comment:
Regarding the second option: excuse me, seems I was wrong. As I understood from documentation and observing some source code, LGBM relies on the encoded values provided by pandas. I.e. in case of categorical column, LGBM will extract integers provided by pandas, but it will not automatically detect this column as categorical for the model unless you explicitly pass it through the `categorical_feature` parameter.